packages/docs-site/src/content/docs/guides/indexers/subquery.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SubQuery
+description: Indexing Taiko with SubQuery
+---
+
+SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports many ecosystems including Taiko, Ethereum, Cosmos, Near, Polygon, Polkadot, Algorand, and more.
+
+Another one of SubQuery's competitive advantages is the ability to aggregate data not only within a chain but across multiple blockchains all within a single project. This allows the creation of feature-rich dashboard analytics and multi-chain block scanners.
+
+## Useful resources:
+
+- [SubQuery Academy (Documentation)](https://academy.subquery.network/)
+- [Taiko Holesky Starter](https://github.com/subquery/ethereum-subql-starter/tree/main/Taiko/taiko-holesky-starter)
+- [Taiko Katla Starter](https://github.com/subquery/ethereum-subql-starter/tree/main/Taiko/taiko-katla-starter)
+- [Quick Start Guide](https://academy.subquery.network/indexer/quickstart/quickstart.html)
+
+For technical questions and support reach out to us `start@subquery.network`
+
+## Running and Hosting Taiko SubQuery APIs
+
+SubQuery is open-source, meaning the developers have the freedom to run it in the following three ways:
+
+- Locally (or a cloud provider), [view the instructions on how to run SubQuery Locally](https://academy.subquery.network/run_publish/run.html).
+- SubQuery's enterprise-level [Managed Service](https://managedservice.subquery.network/), where the SubQuery project will be hosted in production ready services for mission critical data with zero-downtime blue/green deployments. There even is a generous free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).
+- Decentralised [SubQuery Network](https://subquery.network/network), the most open, performant, reliable, and scalable data service for dApp developers. The SubQuery Network indexes and services data to the global community in an incentivised and verifiable way and supports Taiko from launch.
```

Comment:
Provide specific examples of how SubQuery’s decentralized network benefits Taiko, such as improved data availability and robustness.